pxr/imaging/plugin/glfVdb/CMakeLists.txt

```diff
@@ -0,0 +1,49 @@
+set(PXR_PREFIX pxr/imaging)
+set(PXR_PACKAGE glfVdb)
+
+if (NOT ${PXR_ENABLE_GL_SUPPORT})
+    message(STATUS
+            "Skipping ${PXR_PACKAGE} because PXR_ENABLE_GL_SUPPORT is OFF")
+    return()
+endif()
+
+if (PXR_BUILD_OPENVDB_PLUGIN)
+    list(APPEND optionalPublicClasses vdbTextureData)
+    list(APPEND optionalLibs ${OPENEXR_Half_LIBRARY} ${OPENVDB_LIBRARY})
+    list(APPEND optionalIncludeDirs ${OPENVDB_INCLUDE_DIR})
+endif()
```

Comment:
Hey @ablev-dwa -- was just looking at this to scope it out for size, and quickly noticed that maybe we don't need the above 5 lines?

Reply:
Whoops! You are correct, that slipped past me.